src/tokens/Datatoken.ts

```diff
@@ -44,6 +44,12 @@ export interface DispenserParams {
   allowedSwapper?: string // only account that can ask tokens. set address(0) if not required
 }
 
+export interface PublishingMarketFee {
```

Comment:
I think we can move this to `Erc20.ts` file under `@types` folder where we also have `ConsumeMarketFee` interface